homeworks/Vladimir.Karbivnychiy_VladimirSkarb/11-dependency-injection/car-list/src/app/app.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, OnInit} from '@angular/core';
+
+import {Vehicle} from './models';
+import {VehiclesService} from "./services";
+import {finalize} from "rxjs/operators";
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent  implements OnInit{
+
+  ngOnInit() {
+    this.getVehicles();
+  }
+
+  constructor(
+    private vehiclesService: VehiclesService,
+  ) {
+  }
+
+  loader: boolean = true;
+  vehicles: Vehicle[] = [];
+
+  private getVehicles(): any {
+    this.vehiclesService.getCar()
```

Comment:
Little comment:
If in the future you will need to request vehicles again, your loader doesn't work.

So I suggest you add `this.loader = true` at the function start